doc/source/index.rst

```diff
@@ -1,159 +1,31 @@
-
-.. _`on GitHub`: https://github.com/ray-project/ray
-.. _`RaySGD`: raysgd/raysgd.html
-
-.. tip:: Join our `community slack <https://forms.gle/9TSdDYUgxYs8SA9e8>`_ to discuss Ray!
-
-
-Quick Start
------------
-
-First, install Ray with: ``pip install ray``
-
-.. code-block:: python
-
-    # Execute Python functions in parallel.
-
-    import ray
-    ray.init()
-
-    @ray.remote
-    def f(x):
-        return x * x
-
-    futures = [f.remote(i) for i in range(4)]
-    print(ray.get(futures))
-
-To use Ray's actor model:
-
-.. code-block:: python
-
-    import ray
-    ray.init()
-
-    @ray.remote
-    class Counter(object):
-        def __init__(self):
-            self.n = 0
-
-        def increment(self):
-            self.n += 1
-
-        def read(self):
-            return self.n
-
-    counters = [Counter.remote() for i in range(4)]
-    [c.increment.remote() for c in counters]
-    futures = [c.read.remote() for c in counters]
-    print(ray.get(futures))
-
-Visit the `Walkthrough <walkthrough.html>`_ page a more comprehensive overview of Ray features.
-
-Ray programs can run on a single machine, and can also seamlessly scale to large clusters. To execute the above Ray script in the cloud, just download `this configuration file <https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/aws/example-full.yaml>`__, and run:
-
-``ray submit [CLUSTER.YAML] example.py --start``
-
-Read more about `launching clusters <autoscaling.html>`_.
-
-Tune Quick Start
-----------------
-
-`Tune`_ is a library for hyperparameter tuning at any scale. With Tune, you can launch a multi-node distributed hyperparameter sweep in less than 10 lines of code. Tune supports any deep learning framework, including PyTorch, TensorFlow, and Keras.
-
-.. note::
-
-    To run this example, you will need to install the following:
-
-    .. code-block:: bash
-
-        $ pip install ray torch torchvision filelock
-
-
-This example runs a small grid search to train a CNN using PyTorch and Tune.
-
-.. literalinclude:: ../../python/ray/tune/tests/example.py
-   :language: python
-   :start-after: __quick_start_begin__
-   :end-before: __quick_start_end__
-
-If TensorBoard is installed, automatically visualize all trial results:
-
-.. code-block:: bash
-
-    tensorboard --logdir ~/ray_results
-
-.. _`Tune`: tune.html
-
-RLlib Quick Start
+The Ray Community
 -----------------
 
-`RLlib`_ is an open-source library for reinforcement learning built on top of Ray that offers both high scalability and a unified API for a variety of applications.
-
-.. code-block:: bash
-
-  pip install tensorflow  # or tensorflow-gpu
-  pip install ray[rllib]  # also recommended: ray[debug]
-
-.. code-block:: python
+Ray is more than a framework for distributed applications but also an active community of developers, 
+research scientists, and folks that love machine learning. 
```

Comment:
```suggestion
researchers, and folks that love machine learning. 
```

Reply:
for some reason I find the word "folks" here off-putting, but don't have a better suggestion